fusor-ember-cli/app/mirage/config.js

```diff
@@ -56,6 +56,10 @@ export default function() {
     return db.katello_organizations.find(id);
   });
 
+  this.get('/katello/api/v2/organizations/:id/subscriptions', function(db, request) {
+    return {};
```

Comment:
With no subscriptions I get back this JSON, I think the **results** is what  you care about.
```
{
  "organization": {},
  "total": 0,
  "subtotal": 0,
  "page": 1,
  "per_page": 20,
  "error": null,
  "search": null,
  "sort": {
    "by": null,
    "order": null
  },
  "results": []
}
```
When I have subscriptions in the DB, the result looks like this:
```
{
  "organization": {},
  "total": 2,
  "subtotal": 2,
  "page": 1,
  "per_page": 20,
  "error": null,
  "search": null,
  "sort": {
    "by": null,
    "order": null
  },
  "results": [
    {
      "id": 1,
      "cp_id": "ff80808155c0cd1a0155e4760c6200eb",
      "subscription_id": 1,
      "name": "Red Hat Cloud Infrastructure with Smart Management, Premium (2-sockets)",
      "start_date": "2016-03-17T04:00:00.000+0000",
      "end_date": "2017-03-17T03:59:59.000+0000",
      "available": 5,
      "quantity": 5,
      "consumed": 0,
      "account_number": 5530698,
      "contract_number": 10908981,
      "support_level": "Premium",
      "product_id": "RV00007",
      "sockets": 2,
      "cores": null,
      "ram": null,
      "instance_multiplier": 1,
      "stacking_id": 0,
      "multi_entitlement": true,
      "type": "NORMAL",
      "product_name": "Red Hat Cloud Infrastructure with Smart Management, Premium (2-sockets)",
      "unmapped_guest": false,
      "virt_only": false
    },
    {
      "id": 2,
      "cp_id": "ff80808155c0cd1a0155e4760cd8014d",
      "subscription_id": 2,
      "name": "OpenShift Business Partner Self-Supported NFR",
      "start_date": "2016-03-17T04:00:00.000+0000",
      "end_date": "2017-03-17T03:59:59.000+0000",
      "available": 1,
      "quantity": 1,
      "consumed": 0,
      "account_number": 5530698,
      "contract_number": 10908980,
      "support_level": "Self-Support",
      "product_id": "SER0480",
      "sockets": null,
      "cores": null,
      "ram": null,
      "instance_multiplier": 1,
      "stacking_id": null,
      "multi_entitlement": null,
      "type": "NORMAL",
      "product_name": "OpenShift Business Partner Self-Supported NFR",
      "unmapped_guest": false,
      "virt_only": false
    }
  ]
}
```